product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -179,15 +188,15 @@ For more information, see [Periodic maintenance](/biganimal/release/overview/upd
 #### Read-only workloads
 
 !!! Note
-    The **Read-only Workloads** option is available only for primary/standby high-availability cluster types.
+    The **Read-only Workloads** option is available only for primary/standby high-availability and distributed high-availability cluster types.
     
 Enable **Read-only Workloads**. This feature directs read-only operations exclusively toward replicas. If this option is enabled, you might have to raise the IP address resource limits for the cluster:
```

Comment:
This is repeated enough I am wondering if it really needs its own page on increasing IP address resources?